### PR TITLE
UnitTest : String null test for note editor

### PR DESCRIPTION
**Pull request by shrutigitte**
## Purpose / Description
 Unit Test for note editor

## How Has This Been Tested?
Tested locally all tests passed

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/NoteEditorTest.kt [hunk 1]
```diff
@@ -89,6 +89,15 @@ class NoteEditorTest : RobolectricTest() {
         assertThat(actualResourceId, equalTo(CollectionManager.TR.addingTheFirstFieldIsEmpty()))
     }
 
+    @Test
+    fun testErrorNull() = runTest {
+        val noteEditor = getNoteEditorAdding(NoteType.BASIC)
+            .withFirstField("Test")
```
Comment (criticalAY) [inline AnkiDroid/src/test/java/com/ichi2/anki/NoteEditorTest.kt:95]: The error is always null in this case, hence this test it wrong. You need a value to be assigned first then test it

**Review by criticalAY — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution, there are some changes that need to be addressed

**Review by david-allison — APPROVED** [review]
Looks great, cheers!

**Review by criticalAY — APPROVED** [review]
LGTM!

**Review by mikehardy — APPROVED** [review]
Nice

**Pull request merged**